submissions/zhenyakornilov/js-dom/css/styles.css

```diff
@@ -0,0 +1,422 @@
+  text-align: center;
+  height: 5rem;
+  position: relative;
+  overflow-y: hidden;
+}
+
+.header .title {
+  color: #8fe3cf;
+  font-size: 3.2rem;
+}
+
+.header .check-switch {
+  opacity: 0;
+  position: absolute;
+  right: 2rem;
+}
+
+.header .check-switch:checked + .theme-switcher .circle {
+  transform: translatex(2.4rem);
+}
+
+.header .theme-switcher {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #223a51;
+  right: 3rem;
+  width: 5rem;
+  height: 2.6rem;
+  border-radius: 5rem;
+  padding: 0.5rem;
+  right: 1.5rem;
+}
+
+.header .check-switch:focus-visible + .theme-switcher {
+  border: 2px solid #ffc0cb;
+}
+
+.header .check-switch:focus-visible + .theme-switcher .circle {
+  top: 0.05rem;
+  left: 0.05rem;
+  background-color: #ffc0cb;
+}
+
+.header .circle {
+  position: absolute;
+  background-color: #8fe3cf;
+  height: 2.2rem;
+  width: 2.2rem;
+  top: 0.2rem;
+  left: 0.2rem;
+  border-radius: 50%;
+  transition: transform 0.2s linear;
+}
+
+.header .fa-moon,
+.header .fa-sun {
+  color: #f5f5dc;
+}
+
+.dropdown-wrap {
+  display: inline;
+}
+
+.dropdown {
+  display: none;
+  margin-left: 2rem;
+  color: #8fe3cf;
+  padding: 5px 5px 4px 5px;
+}
+
+.dropdown:focus-visible {
+  border: 1px solid #ffc0cb;
+  color: #ffc0cb;
+  border-radius: 5px;
+}
+
+.sidebar.dark {
+  background-color: #041c32;
+}
+
+.sidebar.hidden {
+  visibility: hidden;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  grid-area: sidebar-nav;
+  background-color: #2b4865;
+  overflow-y: hidden;
+}
+
+.sidebar.active {
+  visibility: none;
+}
+
+.menu-items {
+  margin-top: 50px;
+}
+
+.nav-item {
+  border-bottom: 1px solid #f5f5dc;
+  width: 20rem;
+  margin: 10;
+}
+
+.nav-item {
+  height: 2rem;
+  color: #8fe3cf;
+  margin-bottom: 5rem;
+  font-size: 1.8rem;
+}
+
+.nav-item :hover {
+  cursor: pointer;
+  color: #f5f5dc;
+}
+
+.nav-item :focus {
+  color: #ffc0cb;
+}
+
+.nav-item.open {
+  color: #f5f5dc;
+}
+
+.content.dark {
+  background-color: #04293a;
+}
+
+.content {
+  display: flex;
+  grid-area: content-main;
+  background-color: #f5f5dc;
+  position: relative;
+  border-top-left-radius: 15px;
+  border-bottom-left-radius: 15px;
+}
+
+.content-body.dark .museum-desc,
+.content-body.dark .museum-title,
+.content-body.dark .museum-link {
+  color: #f5f5dc;
+}
+
+.content-body.dark .museum-image {
+  filter: brightness(45%);
+}
+
+.content-body {
+  position: relative;
+  margin: 4rem 4rem 0 4rem;
+  display: grid;
+  grid-template-rows: 1fr 4fr 1fr;
+  grid-template-areas: "title title" "desc desc" "more more";
+}
+
+.content-body .museum-title {
+  display: grid;
+  grid-area: title;
+}
+
+.content-body .desc-wrapper {
+  grid-area: desc;
+  line-height: 1.6;
+  font-size: 1.75rem;
+  overflow-wrap: break-word;
+  margin-top: 20px;
+}
+
+.content-body .desc-wrapper .museum-image {
+  float: left;
```

Comment:
`float` is an old stuff and nowadays better to avoid using it, you can use flex instead

Reply:
I used float here to wrap text around image, I haven't figured out how to do that with flex yet.

Reply:
Let's leave it, it is not a big deal. Try to use this tag here, it is a common way of using it, add a caption

Reply:
Did You mean to place museum description to **figcaption** tag here instead of **\<p\>** tag?